docs/ui/components/Collapsible/Collapsible.test.tsx

```diff
@@ -1,17 +1,27 @@
 import { fireEvent, render, screen } from '@testing-library/react';
+import GithubSlugger from 'github-slugger';
 import { PropsWithChildren } from 'react';
 
 import { Collapsible } from '.';
 
+import { HeadingManager, HeadingType } from '~/common/headingManager';
 import { HeadingsContext } from '~/components/page-higher-order/withHeadingManager';
 
+const prepareHeadingManager = () => {
+  const headingManager = new HeadingManager(new GithubSlugger(), { headings: [] });
+  headingManager.addHeading('Base level heading', undefined, {});
+  headingManager.addHeading('Level 3 subheading', 3, {});
+  headingManager.addHeading('Code heading depth 1', 0, {
+    sidebarDepth: 1,
+    sidebarType: HeadingType.InlineCode,
+  });
+
+  return headingManager;
```

Comment:
Small tip - probably in your case you can skip "addHeading" portion (or at least most of them), since tests were working with single slug earlier (since looks like you do not validate link href anyway).